rovlog.c

```diff
@@ -1,19 +1,37 @@
 #include "rovlog.h"
-// You're probably going to want to include other things to use printf() and stuff like that.
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
 
+// You're probably going to want to include other things to use printf() and stuff like that.
```

Comment:
We've taken care of the recommendation in this comment, so to keep things clean we can get rid of this line.